### PR TITLE
[Docenhancement][BotSDKmethods][3846]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Added table for Bot framework SDK methods. Please review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/706490bb91f24baade368b42c5dee43fafdbff90/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2571fdd2-57bf-44c8-b82f-523fd0b948c1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved.

**Comment by surbhigupta12** [comment]
@v-rodriguezd, please resolve conflicts.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0f16c8f122b23d2efd36aad7818cf85c52f0c925/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b6f7b759-70c6-4467-bb60-2ff190c81de7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-rodriguezd** [comment]
> @v-rodriguezd, please resolve conflicts.

Done. Thanks.

File: msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md [hunk 1]
```diff
@@ -238,15 +246,15 @@ The `NotificationSignal` API includes the following response codes:
...
+| **403** | The app is unable to send the signal. 403 response code can occur because of various reasons, such as the tenant admin disables the app and the app is blocked during live site migration. In this case, the payload contains a detailed error message. |
+| **404** | The meeting chat doesn't exist. |
 
 ### Meeting Details API
 
 > [!NOTE]
 > This feature is currently available in [public developer preview](../resources/dev-preview/developer-preview-intro.md) only.
 
-The Meeting Details API enables your app to get static meeting metadata. These are data points that do not change dynamically.
+The Meeting Details API enables your app to get static meeting metadata. The metadata provide data points that don't change dynamically.
```
Comment (v-abirade) [inline msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md:257]: The metadata provides data points that don't change dynamically.

**Review by v-abirade — CHANGES_REQUESTED** [review]
@v-rodriguezd, 
Please make the changes.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e66a89a02b0d5716ec52d51c500170b88b264a1e/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9d7d70a0-c0ff-4777-ab18-4f14fdc501c6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c83b33e50158d290f0adc009b0ff56c384ebba4d/msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md) | 97 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f9a72c72-60ca-48dd-83b1-2f3a03b346fd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved

**Pull request merged**